controllers/apps/secret_to_apimanager_event_mapper.go

```diff
@@ -0,0 +1,58 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	appsv1alpha1 "github.com/3scale/3scale-operator/apis/apps/v1alpha1"
```

Comment:
can we fix imports order here please

Reply:
done, changed a little, but this is how my IDE works in this file...   Thanks